stdlib/Distributed/src/remotecall.jl

```diff
@@ -232,7 +243,7 @@ function del_clients(pairs::Vector)
     end
 end
 
-any_gc_flag = Condition()
+const any_gc_flag = Condition()
```

Comment:
Docs look fine, but I can not comment on this one. Is there a reason this was non-`const`?

Reply:
I can't see why. `Condition` wraps a vector and that's it. There is no other affectation to the `any_gc_flag` in `Distributed`